integration/kubernetes/k8s-sandbox-bindmount.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2022 Apple Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	
+	SYSCONFIG_FILE="/etc/kata-containers/configuration.toml"
```

Comment:
Nit: `KATA_CONFIG_FILE` would be clearer as seeing this I was assuming `/etc/sysconfig/`.